Makefile

```diff
@@ -223,8 +240,10 @@ script/windows-installer/git-lfs-wizard-image.bmp
 # To build a specific release with a custom VERSION suffix, run the following:
 #
 # 	make VERSION=my-version bin/releases/git-lfs-darwin-amd64-my-version.tar.gz
-RELEASE_TARGETS = bin/releases/git-lfs-darwin-amd64-$(VERSION).tar.gz \
+RELEASE_TARGETS = \
```

Comment:
Likewise, I think that we need to add the following here:

- `bin/releases/git-lfs-freebsd-arm64-$(VERSION).tar.gz`
- `bin/releases/git-lfs-windows-arm64-$(VERSION).zip`

Reply:
same

Reply:
dropped unused targets